tests/scripts/gmi.tc.89pct.imagery_clean.sh

```diff
@@ -26,6 +26,7 @@ run_procflow $GEOIPS_TESTDATA_DIR/test_data_gpm/data/1B.GPM.GMI.TB2016.20200917-
              --output_formatter imagery_clean \
              --metadata_filename_formatter metadata_default_fname \
              --metadata_output_formatter metadata_default \
+             --no_sectoring \
```

Comment:
I'm wondering if this should be renamed --no_presectoring?  Might be confusing if you are giving it a sector list, and telling it not to sector?  We're really just telling it not to pre-sector. @jsolbrig

Reply:
Done.